napari/layers/shapes/shapes.py

```diff
@@ -859,8 +859,12 @@ def _extent_data(self) -> np.ndarray:
         if len(self.data) == 0:
             extrema = np.full((2, self.ndim), np.nan)
         else:
-            maxs = np.max([np.max(d, axis=0) for d in self.data], axis=0)
-            mins = np.min([np.min(d, axis=0) for d in self.data], axis=0)
+            maxs = np.max(
+                [d._bounding_box[1] for d in self._data_view.shapes], axis=0
+            )
+            mins = np.min(
+                [d._bounding_box[0] for d in self._data_view.shapes], axis=0
```

Comment:
This is a big change and I think incorrect? `_data_view` only contains the current slice data (?), so any out-of-slice Shapes at the time this is called will not be counted?

Reply:
Nope, just went through the class, and _data_view is just a weird amalgamation of data and other properties. The docstring is definitely incorrect, but I'll leave that battle for another time :joy: